wgpu/src/lib.rs

```diff
@@ -1465,6 +1466,12 @@ pub struct VertexState<'a> {
     /// The name of the entry point in the compiled shader. There must be a function with this name
     /// in the shader.
     pub entry_point: &'a str,
+    /// Specifies the values of pipeline-overridable constants in the shader module module.
+    ///
+    /// The key represents the numeric ID of the constant, if one is specified, and otherwise the constant's identifier name.
+    ///
+    /// The value may represent any of WGSL's concrete scalar types.
+    pub constants: &'a HashMap<String, f64>,
```

Comment:
nitpick: Is there a strong reason to not reexport and use `naga::PipelineConstants` here?

This question applies to other places where this doc has been copy-pasted.

Reply:
I think it would be easier to look at `wgpu`'s docs and see the actual type rather than the alias.

Reply:
Ah, okay. I thought, for some reason, we were re-exporting `naga` API, but if that's not the case, then I don't see a strong reason to do as I suggest.